bot.py

```diff
@@ -76,4 +60,47 @@ async def watchlist_see_all(interaction: nextcord.Interaction):
 
     await interaction.response.send_message(response)
 
+@bot.slash_command(guild_ids=[GUILD_ID], name="watchlist_create", description="create a new watchlist")
+async def watchlist_create(interaction: nextcord.Interaction, arg):
+    #read the json to get all watchlist list
+    watchlist_file = open(WATCHLISTFILENAME, 'r')
+    watchlist_data = json.load(watchlist_file)
+    watchlist_file.close()
+
+    #write in the new watchlist to the json
+    watchlist_data["watchlists"].append({'name': arg, 'media': []})
+    watchlist_file = open(WATCHLISTFILENAME, 'w')
+    watchlist_file.write(json.dumps(watchlist_data))
+    watchlist_file.close()
+
+    response = f"Created a new watchlist named {arg}. Add movies and shows to your new watchlist!"
+    await interaction.response.send_message(response)
+
+@bot.slash_command(guild_ids=[GUILD_ID], name="watchlist_delete", description="delete an existing watchlist")
+async def watchlist_create(interaction: nextcord.Interaction, arg):
```

Comment:
Does this need to be renamed to watchlist_delete? there's two watchlist_create

Reply:
Good find, should be fixed now.